pep-0630.rst

```diff
@@ -424,6 +447,52 @@ For example::
        {NULL},
    }
 
+Module State Access from Slot Methods, Getters and Setters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This is new in Python 3.11.
+
+   .. After adding to limited API:
```

Comment:
This block is commented out, looks like that's not intentional.

Reply:
It is. The function is not in the limited API yet. I wrote the block before the decision to not include it, and didn't want todelete it :)